polaris.shopify.com/pages/examples/tabs-with-actions.tsx

```diff
@@ -0,0 +1,101 @@
+import {LegacyCard, Tabs, TabProps} from '@shopify/polaris';
+import {useState} from 'react';
+import {withPolarisExample} from '../../src/components/PolarisExampleWrapper';
+
+function TabsWithBadgeExample() {
```

Comment:
```suggestion
function TabsWithActionsExample() {
```
Should this be renamed to `TabsWithActionsExample` to match the file name?